loomi/regulariza-diagonal-client
# Feature(NucleusDetail):create Nucleus Timeline

## Feature
- Create Nucleus Timeline
---

**What I did:**

- <thing I’ve done>
  - <detailing>

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/130129111-0b29dec3-0787-4ee4-8900-03671f3324dc.png)
